cli/server.go

```diff
@@ -277,8 +279,8 @@ func server() *cobra.Command {
 				}
 			}
 
-			_, _ = fmt.Fprintf(cmd.ErrOrStderr(), "access-url: %s\n", accessURL)
 			_, _ = fmt.Fprintf(cmd.ErrOrStderr(), "provisioner-daemons: %d\n", provisionerDaemonCount)
+			_, _ = fmt.Fprintf(cmd.ErrOrStderr(), "Access your Coder UI in a browser at: %s\n", accessURL)
```

Comment:
This language is inconsistent with the `provisioner-daemons` line. Thus, I think the original line is better.

Reply:
@ammario We changed this because it wasn't clear to the user what the purpose of the access URL is (e.g., do they launch it in a browser? is it a value they should save for later use?)

Reply:
The same could be argued for the `provisioner-daemons` line. If we want to use that sentence form we should convert the provisioner line as well.